apps/design-system-docs/package.json

```diff
@@ -0,0 +1,52 @@
+{
+	"name": "design-system-docs",
+	"version": "0.0.0",
+	"private": true,
+	"license": "GPL-2.0-or-later",
+	"author": "Automattic Inc.",
+	"description": "Documentation site for the Automattic Design System.",
+	"homepage": "https://github.com/Automattic/wp-calypso/tree/trunk/apps/design-system-docs",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/Automattic/wp-calypso.git",
+		"directory": "apps/design-system-docs"
+	},
+	"scripts": {
+		"docusaurus": "docusaurus",
+		"start": "docusaurus start -p 42987",
+		"build": "docusaurus build",
+		"deploy": "docusaurus deploy",
+		"clear": "docusaurus clear",
+		"serve": "docusaurus serve -p 42987",
+		"write-heading-ids": "docusaurus write-heading-ids",
+		"typecheck": "tsc"
+	},
```

Comment:
Should we also include a `teamcity:build-app` command? This will allow building the app so it can be hosted somewhere directly.

Reply:
Tracked in DS-151